Examples/UIExplorer/js/CameraRollView.js

```diff
@@ -94,6 +94,7 @@ var CameraRollView = React.createClass({
         var imageStyle = [styles.image, {width: imageSize, height: imageSize}];
         return (
           <Image
+            key={asset}
```

Comment:
It is an Image yes, I was following the lead of https://github.com/facebook/react-native/blob/master/Examples/UIExplorer/js/CameraRollExample.js. AFAIK it's fine to use any type as a key but possibly sub-optimal if object references change when their contents don't materially change. Happy to go with path instead if you like - it's probably safe to assume that's unique in this context.